docs/ref/models/querysets.txt

```diff
@@ -3567,14 +3567,11 @@ elsewhere.
 
 .. class:: Q
 
-A ``Q()`` object, like an :class:`~django.db.models.F` object, encapsulates a
-SQL expression in a Python object that can be used in database-related
-operations.
-
-In general, ``Q() objects`` make it possible to define and reuse conditions.
-This permits the :ref:`construction of complex database queries
-<complex-lookups-with-q>` using ``|`` (``OR``) and ``&`` (``AND``) operators;
-in particular, it is not otherwise possible to use ``OR`` in ``QuerySets``.
+A ``Q()`` object represents a SQL condition that can be used in
+database-related operations. It's similar to how an
+:class:`F() <django.db.models.F>` object represents a field value. They make it
```

Comment:
`a field value` -> `the value of a model field or annotated column`.

Reply:
Done, but used "or annotation", since we call them annotations in `.annotate()`.